cluster/acsk.go

```diff
@@ -236,7 +289,8 @@ func CreateACSKClusterFromRequest(request *pkgCluster.CreateClusterRequest, orgI
 	return &cluster, nil
 }
 func (c *ACSKCluster) CreateCluster() error {
-	log.Info("Start create cluster (Alibaba)")
+	c.log.WithField("clusterName", c.modelCluster.Name)
```

Comment:
this is ineffective (WithField returns the contextual logger, and has no side-effect)